Actions/GetWorkflowMultiRunBranches/GetWorkflowMultiRunBranches.ps1

```diff
@@ -0,0 +1,40 @@
+﻿param(
+    [Parameter(Mandatory = $false, HelpMessage = "Comma-separated value of branch names patterns to include if they exist. If not specified, only the current branch is returned. Wildcards are supported.")]
+    [string] $includeBranches
+)
+
+$gitHubHelperPath = Join-Path $PSScriptRoot '../Github-Helper.psm1' -Resolve
+Import-Module $gitHubHelperPath -DisableNameChecking
+
+switch ($env:GITHUB_EVENT_NAME) {
+    'schedule' {
+      Write-Host "Event is schedule: getting branches from settings"
+      $branchPatterns = @($($(ConvertFrom-Json $env:settings).workflowSchedule.includeBranches))
```

Comment:
Add defensive check in case `workflowSchedule` or `includeBranches` is not defined.